openapi.yaml

```diff
@@ -149,5 +169,8 @@ components:
     ToDoBundleResponse:
       allOf:
         - $ref: '#/components/schemas/ToDoResponse'
-      
-  
\ No newline at end of file
+    error400:
```

Comment:
PascalCase ve naming için -> Error400 kullanmaktansa ErrorMessage gibi gitmen lazım bunları tanımlarken sayı kullanmamayı çalış